src/mlpack/methods/ann/layer/transposed_convolution_impl.hpp

```diff
@@ -57,45 +57,118 @@ TransposedConvolution<
     const size_t kernelHeight,
     const size_t strideWidth,
     const size_t strideHeight,
-    const size_t padWidth,
-    const size_t padHeight,
+    const size_t padW,
+    const size_t padH,
     const size_t inputWidth,
     const size_t inputHeight,
     const size_t outputWidth,
-    const size_t outputHeight) :
+    const size_t outputHeight,
+    const std::string& paddingType) :
+    TransposedConvolution(
+    inSize,
+    outSize,
+    kernelWidth,
+    kernelHeight,
+    strideWidth,
+    strideHeight,
+    std::tuple<size_t, size_t>(padW, padW),
+    std::tuple<size_t, size_t>(padH, padH),
+    inputWidth,
+    inputHeight,
+    outputWidth,
+    outputHeight,
+    paddingType)
```

Comment:
Should I indent these lines?

Reply:
Yes please indent here with 2 tabs. 
https://github.com/mlpack/mlpack/wiki/DesignGuidelines#constructor-initialization-lists

Reply:
and same for other places

Reply:
I have indented the initialization list, and fixed the comment space issue. Thanks.